Doppler.HtmlEditorApi/Domain/CampaignState.cs

```diff
@@ -2,7 +2,19 @@
 
 namespace Doppler.HtmlEditorApi.Domain;
 
-public record CampaignState(bool OwnCampaignExists, bool ContentExists, int? EditorType, CampaignStatus? CampaignStatus)
+public enum TestABCondition
+{
+    NoTestAB = 0,
+    TypeTestABSubject = 1,
+    TypeTestABContent = 2
+}
+
+public record CampaignState(
```

Comment:
Could you make it abstract to force us to use a predefined type for testAB Campaigns?

```suggestion
public abstract record CampaignState(
```

Reply:
Related comments:

* https://github.com/FromDoppler/doppler-html-editor-api/pull/113/files#r872347266 
* https://github.com/FromDoppler/doppler-html-editor-api/pull/113/files#r872346834

Reply:
Done